apps/trading/components/accounts-container/sidebar-accounts-container.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import { useT } from '../../lib/use-t';
+import { SwapContainer } from '../swap';
+import { WithdrawContainer } from '../withdraw-container';
+import {
+  AccountsContainer,
+  type AccountsContainerProps,
+} from '../accounts-container/accounts-container';
+import React from 'react';
+import { DepositContainer } from '@vegaprotocol/deposits';
+import { TransferContainer } from '@vegaprotocol/accounts';
+import { VegaIcon, VegaIconNames } from '@vegaprotocol/ui-toolkit';
+
+enum View {
+  Deposit = 'Deposit',
+  Swap = 'Swap',
+  Transfer = 'Transfer',
+  Withdraw = 'Withdraw',
+}
+
+type InnerView = [view: View, assetId: string];
+
+export const SidebarAccountsContainer = ({
+  pinnedAssets,
+}: Pick<AccountsContainerProps, 'pinnedAssets'>) => {
+  const t = useT();
+  const [innerView, setInnerView] = useState<InnerView | undefined>(undefined);
+
+  if (innerView) {
+    const [view] = innerView;
+    return (
+      <div className="p-2">
+        <header className="text-lg">
+          <button
+            className="flex gap-1 items-center"
+            onClick={() => {
+              setInnerView(undefined);
+            }}
+          >
+            <span className="text-muted inline-flex items-center">
+              <VegaIcon name={VegaIconNames.CHEVRON_LEFT} />
+            </span>
+            <span>{t(view)}</span>
+          </button>
+        </header>
+        <div className="mt-2">
+          <InnerContainer innerView={innerView} setInnerView={setInnerView} />
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <AccountsContainer
+      pinnedAssets={pinnedAssets}
+      orderByBalance
+      hideZeroBalance
+      onClickDeposit={(assetId) => {
+        setInnerView([View.Deposit, assetId]);
+      }}
+      onClickSwap={(assetId) => {
+        setInnerView([View.Swap, assetId]);
+      }}
+      onClickTransfer={(assetId) => {
+        setInnerView([View.Transfer, assetId]);
+      }}
+      onClickWithdraw={(assetId) => {
+        setInnerView([View.Withdraw, assetId]);
+      }}
+    />
+  );
+};
+
+const InnerContainer = ({
+  innerView,
+  setInnerView,
+}: {
+  innerView: InnerView;
+  setInnerView?: (innerView: InnerView | undefined) => void;
+}) => {
+  const [view, assetId] = innerView;
+  switch (view) {
+    case View.Deposit:
+      return <DepositContainer assetId={assetId} />;
+
+    case View.Swap:
+      return (
+        <SwapContainer
```

Comment:
Swap container renders a title, so we end up with an awkward double title. Can you remove the title from the container, and then add the title on the standalone swap page.

It would be good to check all the titles rendered within these components and make sure everything looks consistent

Reply:
Removed the title completely, we don't show the additional title for any of the containers so it doesn't make sense IMO to have it only for swap. Also moved the "Go to market" link to the bottom of the form.